builds/misc/templates/install-dotnet3.yaml

```diff
@@ -1,6 +0,0 @@
-steps:
-  - task: UseDotNet@2
-    displayName: Install .NET Core sdk
```

Comment:
The advantage to putting it in a template is that you only have to change it in one place. But whatever you decide is fine.